python/grass/temporal/temporal_vector_algebra.py

```diff
@@ -314,14 +314,14 @@ def overlay_cmd_value(self, map_i, tbrelations, function, topolist=["EQUAL"]):
         mapainput = map_i.get_id()
         # Append command list of given map to result command list.
         if "cmd_list" in dir(map_i):
-            resultlist = resultlist + map_i.cmd_list
+            resultlist += map_i.cmd_list
         for topo in topolist:
             if topo.upper() in tbrelations.keys():
                 relationmaplist = tbrelations[topo.upper()]
                 for relationmap in relationmaplist:
                     # Append command list of given map to result command list.
                     if "cmd_list" in dir(relationmap):
-                        resultlist = resultlist + relationmap.cmd_list
+                        resultlist += relationmap.cmd_list
```

Comment:
These two changes here are the only ones that I'm a little less sure, as if you look at the doc page for this ruff rule, it shows an example of what can happen with mutable types, like lists, with an augmented assignment.
We aren't in the case that `resultlist` is assigned to a variable, but later augmented, but rather using `map_i.cmd_list` to augment `resultlist`, and then at the end `resultlist` is assigned to `map_i.cmd_list`.

I would just like a little second opinion on this.

Reply:
Seems fine to me. I ran some manual tests but could not find any issue with this augmentation...

Reply:
I wasn't that scared too. Fine then!